Nim/Nim/Program.cs

```diff
@@ -0,0 +1,121 @@
+﻿Console.WriteLine("Welcome to Nim!");
+Console.WriteLine("1 Player(1)");
+Console.WriteLine("2 Players(2)");
+string player = Console.ReadLine();
+int intPlayer = Convert.ToInt32(player);
+
+if (intPlayer == 1)
+{
+    int matches = 25;
+    int loop = 0;
+    Start:
+    if (loop < matches)
+    {
+        Console.Write("|");
+        loop++;
+        goto Start;
+    }
+    Number:
+    loop = 0;
+    Console.WriteLine("");
+    Console.WriteLine($"How many matches would you like to draw?(1, 2 or 3)({matches})");
+    string draw = Console.ReadLine();
+    int intDraw = Convert.ToInt32(draw);
+    if (intDraw == 1 || intDraw == 2 || intDraw == 3)
```

Comment:
You could use some smarter conditions here: if it is greater than 0 and it is less than 4